lib/AfricasTalking/Payments.rb

```diff
@@ -76,7 +76,10 @@ def mobileB2B options
 			'amount'             => options['amount'],
 			'metadata'           => options['metadata']
 		}
-		url      = getMobilePaymentB2BUrl()   
+		url = getMobilePaymentB2BUrl()
+        if options['requester'] != nil
```

Comment:
This is a more readable code.
```suggestion
       parameters['requester'] = options['requester'] if options['requester'] != nil
```

Reply:
Thanks @babajidemm replaced this with https://ruby-doc.org/core-2.5.1/Hash.html#method-i-key-3F method.